lib/service/connect.go

```diff
@@ -55,8 +55,9 @@ func (process *TeleportProcess) reconnectToAuthService(role teleport.Role) (*Con
 			// if connected and client is present, make sure the connector's
 			// client works, by using call that should succeed at all times
 			if connector.Client != nil {
-				_, err = connector.Client.GetNamespace(defaults.Namespace)
+				pingResponse, err := connector.Client.Ping(process.ExitContext())
 				if err == nil {
+					process.setClusterFeatures(pingResponse.GetServerFeatures())
```

Comment:
Could you please log server features loaded from auth server here? In Info mode. It would be helpful for troubleshooting, right now there's no visibility.

Reply:
logged